R/bias.R

```diff
@@ -206,6 +199,17 @@ bias_quantile <- function(predictions, quantiles, true_value) {
   return(bias)
 }
 
+# quantiles must be between 0 and 1, increase, and be unique
```

Comment:
We might want to move this to check_input_helpers.R and add some documentation around it. But can also do it next time when we rework all input checks as suggested in #276

Reply:
I'd suggest doing in #276

Reply:
I ended up doing this here because why not and Claude was happy to help.